prometheus.spec

```diff
@@ -0,0 +1,115 @@
+%define debug_package %{nil}
+
+Name:           prometheus
+Version:        %{version}
+%if 0%{use_systemd}
+Release:        1%{?dist}systemd
+Requires:       systemd-units
+%else
+Release:        1%{?dist}nosystemd
+Requires:       daemonize
+%endif
+Summary:        Prometheus is a systems and service monitoring system. It collects metrics from configured targets at given intervals, evaluates rule expressions, displays the results, and can trigger alerts if some condition is observed to be true.
+Group:          System Environment/Daemons
+License:        See the LICENSE file at github.
+URL:            https://github.com/prometheus/prometheus
+Requires(pre):  /usr/sbin/useradd
+AutoReqProv:    No
+
+%description
+
+Prometheus is a systems and service monitoring system.
+It collects metrics from configured targets at given intervals, evaluates
+rule expressions, displays the results, and can trigger alerts if
+some condition is observed to be true.
+
+%prep
+
+%build
+
+%install
+install -d \
+	var/log/prometheus \
+	var/run/prometheus \
+	var/lib/prometheus \
+	usr/bin \
+	etc/prometheus \
+	etc/sysconfig \
+	usr/share/prometheus \
+	usr/share/prometheus/consoles \
+	usr/share/prometheus/console_libraries
+install -m 644 "%{src_root}/rpm/prometheus.sysconfig" etc/sysconfig/prometheus
+install -m 644 "%{src_root}/rpm/prometheus.yaml" etc/prometheus/prometheus.yaml
+install -m 755 "%{src_root}/prometheus" usr/bin/prometheus
+install -m 755 "%{src_root}/promtool" usr/bin/promtool
+
+%if 0%{use_systemd}
+install -d lib/systemd/system
+install -m 644 "%{src_root}/rpm/prometheus.service" lib/systemd/system/prometheus.service
+%else
+install -d etc/init.d
+install -m 755 "%{src_root}/rpm/prometheus.init" etc/init.d/prometheus
+install -d etc/logrotate.d
+install -m 644 "%{src_root}/rpm/prometheus.logrotate" etc/logrotate.d/prometheus
```

Comment:
No, if we are not using systemd, then we install SysV init-script:
```
$ grep LOG_FILE rpm/prometheus.init 
LOG_FILE=/var/log/prometheus/prometheus.log
  daemonize -u ${DAEMON_USER} -p ${PID_FILE} -l ${LOCK_FILE} -a -e ${LOG_FILE} -o ${LOG_FILE} ${DAEMON} ${ARGS}
```
And if we are using systemd, then logs managed by systemd.